qa/suites/rados/basic/tasks/rados_api_tests.yaml

```diff
@@ -12,6 +12,7 @@ overrides:
     - \(PG_AVAILABILITY\)
     - \(PG_DEGRADED\)
     - CEPHADM_STRAY_DAEMON
+    - do not have an application enabled
```

Comment:
This may not be sufficient, see https://github.com/ceph/ceph/commit/151364498361acb4f7ffeabb714ff62f46a61960.

Reply:
Fixed